src/umap/Buffer.cpp

```diff
@@ -345,7 +346,9 @@ Buffer::Buffer( void )
 }
 
 Buffer::~Buffer( void ) {
-  UMAP_LOG(Debug, m_stats);
+#ifdef UMAP_DISPLAY_STATS
```

Comment:
How do you control whether stats are collected during the run? I was looking for code that would make sure logging was enabled if UMAP_DISPLAY_STATS was turned on, and didn't see that code.

Reply:
Stats are collected always.  This only controls if they are printed.